package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "clean": "rm -rf node_modules && yarn",
+    "clean": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && yarn && rm ios/Podfile.lock && yarn install:pod && react-native start --resetCache",
```

Comment:
not sure we want the start at the end but I have that alias almost exactly. With an addition to clean the android project and delete derived data

Reply:
Should we add the Android stuff as well?

Reply:
It would be running `./gradlew clean` inside the android folder. I don't see why not.

Reply:
You could add `./android/gradlew -p ./android clean` for Android